examples/interactive-script.html

```diff
@@ -226,9 +240,20 @@
 
         function executeScript() {
             document.getElementById("execute-script").disabled = true;
+            let ipfsDirectoryInput = document.getElementById("ipfs-directory-hash");
+            ipfsDirectoryInput.disabled = true;
+
+            let webSocketUrl = "ws://localhost:8080/script/interactive/" + scriptId;
+            let ipfsDirectoryHash = ipfsDirectoryInput.value;
```

Comment:
```suggestion
            let ipfsDirectoryHash = ipfsDirectoryInput.value.trim();
```
So you don't need to trim it 2 times below.

Reply:
I looked at it more closely, calling `.trim()` here might throw null pointer exception since this is JavaScript, not Kotlin (have a look at the if statement below).